infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/DeserializableListTypeDefinition.java

```diff
@@ -26,4 +26,8 @@ public DeserializableListTypeDefinition(
       final Optional<Integer> maxItems) {
     super(itemType, Function.identity(), minItems, maxItems);
   }
+
+  public DeserializableListTypeDefinition(final DeserializableTypeDefinition<T> itemType) {
```

Comment:
is this actually used?

Reply:
Yeah, in the Archiver

Reply:
weird, intellij wasn't finding any reference

Reply:
you are right, it's not, removed